packages/loot-design/src/svg/Add.mobile.js

```diff
@@ -1,24 +1,7 @@
 /* This file is auto-generated, do not touch! Please edit the SVG file instead. */
 import React from "react";
 
-import {
-  Svg,
-  Circle,
-  Ellipse,
-  G,
-  LinearGradient,
-  RadialGradient,
-  Line,
-  Path,
-  Polygon,
-  Polyline,
-  Rect,
-  Symbol,
-  Text,
-  Use,
-  Defs,
-  Stop
-} from "mobile/node_modules/react-native-svg";
+import { Svg, Path } from "mobile/node_modules/react-native-svg";
```

Comment:
I'm not sure about these svg ones. These svg files are automatically generated and all the available components are imported because we don't know which ones are actually used when generating the file. These changes will be overwritten next time we run the script to generate svgs.

Can we ignore this rule in the svg folder?

Reply:
Yeah, we could add a `/* eslint-ignore rule-name */` line into the output of the generation script to avoid this but I think it would be nicer to have these extra imports automatically pruned by linting after generation.

That said, the linting setup I used for this is on a branch that suffers from the issue described in #127 so I think we can cross that bridge when we come to it. No rules are currently being enforced on this package by CI so we can leave it as is.